src/routes/+layout.svelte

```
@@ -6,8 +6,8 @@
 	import "@fontsource/fira-code"
 	import "$lib/crc32"
 
-	import { getCurrentWebviewWindow } from "@tauri-apps/api/webviewWindow"
-	import { ComposedModal, ModalBody, ModalFooter, ModalHeader, SkipToContent, ToastNotification } from "carbon-components-svelte"
+	import { appWindow } from "@tauri-apps/api/window"
+	import { ComposedModal, HeaderNavItem, ModalBody, ModalFooter, ModalHeader, SkipToContent, ToastNotification } from "carbon-components-svelte"
```

ESLint

[error] 'HeaderNavItem' is defined but never used.

(@typescript-eslint/no-unused-vars)